yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -351,14 +351,6 @@ private[spark] class Client(
 
     val statCache: Map[URI, FileStatus] = HashMap[URI, FileStatus]()
 
-    val oldLog4jConf = Option(System.getenv("SPARK_LOG4J_CONF"))
```

Comment:
Here I removed the support of `SPARK_LOG4J_CONF`, though I already did it in #11603 , I can handle the merge conflicts.